tests/settings_tests/tests.py

```diff
@@ -441,3 +441,130 @@ def test_tuple_settings(self):
             finally:
                 del sys.modules['fake_settings_module']
                 delattr(settings_module, setting)
+
+
+class SettingChangeEnterException(Exception):
+    pass
+
+
+class SettingChangeExitException(Exception):
+    pass
+
+
+class OverrideSettingsIsolationOnExceptionTest(SimpleTestCase):
+    """
+    ``override_settings`` context manager should restore settings
+    when at least one of the receivers of "setting_changed" signal fails.
+
+    Do this with three cases of failure:
+
+    1) receiver fails when the signal is sent while both entering and exiting the context manager,
+    2) receiver fails on enter,
+    3) receiver fails on exit.
+
+    Make sure that in each case ALL receivers are called when exiting the context manager.
+    """
+
```

Comment:
Minor point: There are a couple of these blank lines after docstrings that I'm not sure about.

Reply:
Not sure, what other linebreaks do you mean.